### DISCUSSION TITLE
Increment a value in a table.

### DISCUSSION THREAD

**Discussion by Shaun-Regenbaum** [supabase/supabase#1624] [Questions] [answered]
How can I use update() to just increment a int in the database by one?
Is it possible without first making a select call to grab the value from the database?

**Comment by inian** [comment, marked as answer]
Check this previous discussion on this Shaun https://github.com/supabase/supabase/discussions/909#discussioncomment-516407

**Reply by Shaun-Regenbaum** [reply]
As a follow up for future peeps, do I need to modify this query in any way to make sure it specifies a specific column?
create function increment () 
returns void as
$$
  update table_name 
  set field_name = field_name + 1
$$ 
language sql volatile;
